scripts/run_plier.R

```diff
@@ -0,0 +1,74 @@
+suppressPackageStartupMessages(library(devtools))
+suppressPackageStartupMessages(library(argparse))
+
+# install PLIER from GitHub if not already installed (PLIER
+# is not available through Conda)
+suppressPackageStartupMessages(install_github('wgmao/PLIER'))
+suppressPackageStartupMessages(library(PLIER))
+
+process_data <- function(data) {
+    # adapted from code at:
+    # https://github.com/gitter-lab/prmf/blob/devel/script/PLIER/PLIER_wrapper.R#L61
+
+    # PLIER requires that all genes have a measurement, so
+    # filter for rows that meet this requirement
+    data_dim <- dim(data)
+    data_rowsums <- rowSums(data)
+    inds <- which(data_rowsums != 0)
+    if (length(inds) < data_dim[1]) {
+        warning(paste0("Subsetting data matrix to remove rows without measurements: ",
+                       data_dim[1], " -> ", length(inds)))
+        data <- data[inds, ]
+        data_dim <- dim(data)
```

Comment:
Is there a reason to reassign `data_dim` if it's not returned from the function?

Reply:
No good reason (left over from some code that I copy/pasted)